Week-1/Blastoise/Web_scrapper.py

```diff
@@ -0,0 +1,127 @@
+import requests
+from bs4 import BeautifulSoup as soup
+import json
+import smtplib
+import csv
+import datetime
+from threading import Timer
+# For running code every minute
+def runn_multiple_times():
+    # For putting date and time of entry in the "price_at_each_run.csv"
+    date_time=datetime.datetime.now()
+    
+    #Parsing information of both the phones
+    iphone_url=requests.get("https://www.flipkart.com/apple-iphone-11-white-128-gb/p/itm846a8ba21ab11?pid=MOBFKCTS4MZVJ6FJ&lid=LSTMOBFKCTS4MZVJ6FJPY4SUX&marketplace=FLIPKART&srno=s_1_1&otracker=AS_Query_OrganicAutoSuggest_2_6_na_na_na&otracker1=AS_Query_OrganicAutoSuggest_2_6_na_na_na&fm=SEARCH&iid=1faeec82-078c-4afb-ac82-005cd00f7df2.MOBFKCTS4MZVJ6FJ.SEARCH&ppt=sp&ppn=sp&ssid=r77hztg9jk0000001576406173066&qH=3422ffa2c9fac114")
+    iphone_content=iphone_url.content
+    iphone_parser=soup(iphone_content,'html.parser')
+    oneplus_url=requests.get("https://www.flipkart.com/oneplus-7t-pro-haze-blue-256-gb/p/itm0ce470755470d?pid=MOBFHC8GY8XRXJPF&lid=LSTMOBFHC8GY8XRXJPFHKJLSY&marketplace=FLIPKART&srno=s_1_1&otracker=AS_QueryStore_OrganicAutoSuggest_1_12_na_na_pr&otracker1=AS_QueryStore_OrganicAutoSuggest_1_12_na_na_pr&fm=SEARCH&iid=50b91227-2253-4339-8e22-436dc4221d50.MOBFHC8GY8XRXJPF.SEARCH&ppt=sp&ppn=sp&ssid=x6yuv8m01s0000001576410834553&qH=5e92b45a5bf6c752")
+    oneplus_content=oneplus_url.content
+    oneplus_parser=soup(oneplus_content,'html.parser')
+    
+    #Adding the Properties of Iphone
+    price_iphone='Price'
+    feat_name_iphone=iphone_parser.findAll("td",{"class":"_3-wDH3 col col-3-12"})
+    feat_name_iphone_edit=[]
+    feat_name_iphone_edit.append(price_iphone)
+    for i in feat_name_iphone:
```

Comment:
Please refrain from using single character variables.

Reply:
I made it so that it will be easier for review.

Reply:
I'm not sure how a single character variable will help me understand your code better.

Reply:
Sorry i miss understood.I will try to remove them.